### PR TITLE
Add Redisinsight v1.12.1 release notes.

### PR DESCRIPTION
**Pull request by GnaneshKunal**

### CODE REVIEW

**Comment by GnaneshKunal** [comment]
This PR can be merged. RedisInsight v1.12.1 was just released.

File: content/ri/release-notes/v1.12.0.md [hunk 1]
```diff
@@ -6,12 +6,30 @@ description: RedisInsight v1.12.0
 weight: 2
 ---
 
+## 1.12.1 (July 2022)
+
+This is the maintenance release of RedisInsight 1.12 (v1.12.1)!
+
+### Critical Bug Fix:
+- Core:
+  - When adding or removing a Redis Enterprise Software or Redis Enterprise Cloud database in RedsInsight v1 that has the RediSearch module loaded, all hashes within that database would be deleted.
```
Comment (nermiller) [inline content/ri/release-notes/v1.12.0.md:15]:
```suggestion
  - When you add or remove a Redis Enterprise Software or Redis Enterprise Cloud database in RedsInsight v1 that has the RediSearch module loaded, all hashes within that database are deleted.
```

File: content/ri/release-notes/v1.12.0.md [hunk 2]
```diff
@@ -6,12 +6,30 @@ description: RedisInsight v1.12.0
...
+
+### Fixes:
+- Core:
+  - Added curl command to container.
+  - Fixed container vulnerabilities (CVE-2022-1292, CVE-2022-2068).
+- RediSearch:
+  - Fixed bug where index info doesn't report right information for RediSearch v2.
+- Profiling:
+  - Added support for profiling MODULE commands.
```
Comment (nermiller) [inline content/ri/release-notes/v1.12.0.md:24]:
```suggestion
  - Added support for profiling module commands.
```

File: content/ri/release-notes/v1.12.0.md [hunk 3]
```diff
@@ -6,12 +6,30 @@ description: RedisInsight v1.12.0
...
+### Critical Bug Fix:
+- Core:
+  - When adding or removing a Redis Enterprise Software or Redis Enterprise Cloud database in RedsInsight v1 that has the RediSearch module loaded, all hashes within that database would be deleted.
+
+### Fixes:
+- Core:
+  - Added curl command to container.
+  - Fixed container vulnerabilities (CVE-2022-1292, CVE-2022-2068).
+- RediSearch:
+  - Fixed bug where index info doesn't report right information for RediSearch v2.
```
Comment (nermiller) [inline content/ri/release-notes/v1.12.0.md:22]:
```suggestion
  - Fixed index info to report correct information for RediSearch v2.
```

File: content/ri/release-notes/v1.12.0.md [hunk 4]
```diff
@@ -6,12 +6,30 @@ description: RedisInsight v1.12.0
...
+- Profiling:
+  - Added support for profiling MODULE commands.
+  - Added support for viewing information of clients using ipv6 addresses.
```
Comment (nermiller) [inline content/ri/release-notes/v1.12.0.md:25]:
```suggestion
  - Added support for viewing information of clients that use IPv6 addresses.
```

File: content/ri/release-notes/v1.12.0.md [hunk 5]
```diff
@@ -6,12 +6,30 @@ description: RedisInsight v1.12.0
...
+- RedisGraph:
+  - Added support for `RO_QUERY` only mode. Now RedisGraph can work with just `RO_QUERY` command.
```
Comment (nermiller) [inline content/ri/release-notes/v1.12.0.md:27]:
```suggestion
  - Added support for `RO_QUERY` only mode. RedisGraph now responds to the `RO_QUERY` command.
```

**Review by nermiller — APPROVED** [review]
Just a couple of sentence edits.

**Comment by GnaneshKunal** [comment]
@nermiller Addressed all comments

**Pull request merged**